ccd-definition/CaseEvent/Camunda/DocmosisEvents-nonprod.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "GENERATE_RESPONSE_SEALED",
+    "Name": "Generate response sealed form",
+    "Description": "Generates sealed form for claim response",
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/about-to-submit",
```

Comment:
this should be -
"CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/cases/callbacks/version/V_1/about-to-submit",

V1 is missing here

Reply:
For this one, there is new fields adding in CaseField-CUI-nonprod.json and related functions in civil-service, which may cause error in functional test because the CUI-nonprod.json things should be excluded in other environment. Therefore, as advice from Sampan, we need to create a DocmosisEvents-CUI-nonprod.json to prevent this happen.